### DISCUSSION TITLE
[Projects Beta] Single and multi select

### DISCUSSION THREAD

**Discussion by amadeogallardo** [community/community#16519] [Projects and Issues]
It would be great to:

- Allow more than 25 options on single-select fields.
- Allow multi-select fields.

**Comment by amadeogallardo** [comment]
Seems like the GitHub team updated the max options to 50 items. It would be great to have this limit a bit higher though. 

The multi-select field support is still not ready.

**Comment by MPLew-is** [comment]
Multi-select fields would be a huge improvement - our only option to replicate this functionality right now are issue labels, but there's no option to customize _which_ labels get shown on a board so they all blend together.
